src/renderer/Components/CreatingPackage/CreatingOrder/BookmarkToolbar/index.jsx

```diff
@@ -116,6 +141,23 @@ function BookmarkToolbar() {
           </button>
         </Modal>
       )}
+      {
+        <InfoModal isOpen={isInfoModalOpen}>
```

Comment:
해당 <InfoModal> 컴포넌트는 부모 컴포넌트가 언마운트 됐다 마운트 될 때마다 항상 불필요한 리랜더링이 발생할 것 같습니다.
필요한 경우에 대해서만 랜더링 시켜주는 것은 어떻게 생각하시나여? 🤔 😁
```suggestion
      { isInfoModalOpen &&
        <InfoModal isOpen={isInfoModalOpen}>
```

Reply:
InfoModal 컴포넌트에
```
function InfoModal({ isOpen, children }) {
  if (!isOpen) return null;
```
이렇게 코드를 써뒀고
현재 `isInfoModalOpen` 의 초기값은 false인 상태인데 이런경우에도 리렌더링 부분에서 문제가 될까요?😶

Reply:
아! 얼리리턴의 경우 제가 생각했을때 null 값을 리턴하게 되므로 리렌더링에 문제가 될 것 같지 않습니다 ~!😁